### PR TITLE
Interactive Presentation + new projects arborescence

### PR DESCRIPTION
**Pull request by rbadr**
-  multi-project arborescence
- feat Interactive Presentation

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS question_reponse;
```
Comment (rouvoy) [inline database.sql]: What is the purpose of this file at the root of the project that insert these questions in the database?

**Comment by rouvoy** [comment]
The code clearly lacks of documentation.

File: database.sql (hunk 1 above)
Comment (rbadr) [reply database.sql]: It's used in the presentation example, it definitely shouldn't be in the root of the project, we'll move it to the right folder.

**Comment by rbadr** [comment]
Yes I agree, we're still working on cleaning and providing a good documentation within the code. We'll update this pull request during the week.

**Pull request merged**